www/src/pages/updates/notes/2022-02-11.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: February 11, 2022
+---
+
+Welcome to 2022, and welcome to our new updated format! This new structure intends to provide more granularity and focus on the significant changes to Thumbprint. We'll focus on three main categories of "What's new," "Updates and changes," and "What's coming up."
+
+## 🎉 What’s new
+
+It’s always exciting to get a new pair of shoes. While we won’t don these changes on our feet, these improvements to the product are even more exciting than those new sneakers!
+
+There are two new additions to our icon family.
+
+1. circle-map-pin-filled
+1. circle-money-filled
+
+## 🛠 Updates and changes
+
+We know Thumbprint is a great product, but there is always room for improvement, just like any home renovation project. Here are some updates and changes to the product that we’re excited to celebrate!
+
+### 1. Breakpoint sizing in guidelines
+
+The specific breakpoint values have not been added to the [Breakpoints Guidelines resouce](https://thumbprint.design/guide/product/breakpoints/). You can now find out exactly where the Small, Medium, and Large responsive breakpoints will be triggered. Documentation is even available from both the engineering and design perspective.
```

Comment:
```suggestion
The specific breakpoint values have not been added to the [Breakpoints Guidelines resouce](/guide/product/breakpoints/). You can now find out exactly where the Small, Medium, and Large responsive breakpoints will be triggered. Documentation is even available from both the engineering and design perspective.
```

It's better to not include the domain so that the links works even in the deploy previews.